components/Gamecard/index.js

```diff
@@ -1,27 +1,44 @@
 import { StyledGameCard } from "./Gamecard.styled";
 import { StyledImage } from "./Gamecard.styled";
 import Link from "next/link";
+import ToggleLibraryStateButton from "../ToggleLibraryStateButton";
+import ToggleWishlistStateButton from "../ToggleWishlistStateButton";
+import DetailsPageButton from "../DetailsPageButton";
+import { StyledDiv } from "./Gamecard.styled";
 
-export default function Gamecard({ games }) {
+export default function Gamecard({
+  games,
+  onToggleLibraryClick,
+  onToggleWishlistClick,
+}) {
   return (
     <>
       <ul>
         {games.map((game) => (
-          <Link href={`/games/${game.id}`} key={game.id}>
-            <StyledGameCard>
+          <StyledGameCard key={game.id}>
+            <Link href={`/games/${game.id}`}>
               <h3>{game.title}</h3>
               <StyledImage
                 src={game.imgpath}
                 alt={"gamecover"}
```

Comment:
Instead of `"gamecover"` you could use something like `game.title` to put the description more in relation to the image.
```suggestion
                alt={game.title}
```